Resources/Audio/Effects/Grenades/SelfDestruct/attributions.yml

```
@@ -0,0 +1,7 @@
+- files:
+    - SDS_Charge2.ogg
+  license: CC-BY-SA-3.0
+  source: https://freesound.org/people/Teh_Bucket/sounds/518739/
+  # couldn't figure out how to source multiple (the right way) without shit breaking so heres the rest: https://pixabay.com/sound-effects/switchbigpowerwav-14710/ https://pixabay.com/sound-effects/shield-recharging-107016/
```

```suggestion
  license: Custom
  source: https://freesound.org/people/Teh_Bucket/sounds/518739/
  # couldn't figure out how to source multiple (the right way) without shit breaking so heres the rest: https://pixabay.com/sound-effects/switchbigpowerwav-14710/ https://pixabay.com/sound-effects/shield-recharging-107016/
```

CC 3.0 is incorrect as pixabay is not licensed under that.